ios/ZingoMobile.xcodeproj/project.pbxproj

```diff
@@ -543,7 +543,7 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.ZingoMobile;
+				PRODUCT_BUNDLE_IDENTIFIER = com.ZingoLabs.Zingo;
```

Comment:
Is it possible for this to be:

`org.ZingoLabs.Zingo`

?

Reply:
This is so frustrating... honestly. Because this is impossible to change now. I have to delete the App, the certificate, profile, etc... and start over again.... arghhhhhh

Reply:
If you want to change this... I have to remove the entire App, and start over again... FYI

Reply:
Oh..   well have both domains, so let's not do that extra work!   Unless you already did!